### ISSUE TITLE
Keyboard navigation isn't possible on Safari and on Chrome on MacOS

### ISSUE THREAD

**Issue by sv-hyacoub** [mozilla/blurts-server#1600]
* Affected versions
Chrome Version 80.0.3987.132 (Official Build) (64-bit)
Safari Version 13.0.5

* Affected platforms:
MacOS 10.14

* Steps to reproduce
1. Go to https://blurts-server-review.herokuapp.com
2. Enter any valid email address with breaches and click on "Check for Breaches".
3. Try to reach "Monitor Sign Up Promo" banner using the keyboard

* Expected behaviour:
Keyboard navigation should be possible on Safari and on Chrome on MacOS

* Actual behaviour:
Keyboard navigation isn't possible on Safari and on Chrome on MacOS

**Comment by EMMLynch** [comment]
Closing since we've redesigned the site and functionality since this was created. If you feel that this is still needed, please let me know.
